mirrord/cli/src/main.rs

```diff
@@ -321,6 +322,12 @@ async fn exec(args: &ExecArgs, watch: drain::Watch) -> Result<()> {
         args.binary, args.binary_args
     );
 
+    let container_detection = Regex::new(".*(docker|podman|nerdctl).*")
```

Comment:
```suggestion
    let container_detection = Regex::new("docker|podman|nerdctl")
```
Should be the same (doesn't have to match full string)